main.py

```diff
@@ -80,65 +86,73 @@ def update_convert_upload(self):
         parent_patches : Set[str] = set()
         # First separate between parents and replies. All parents of patchsets will be uploaded
         parents, replies = self.split_parent_and_reply_messages(new_messages)
-
+        
         for message in parents:
             parent_patches.add(message.id)
             messages_to_upload.append(message.id)
-
+        
         # Determine which of the replies should be uploaded
         for message in replies:
             if message.in_reply_to in parent_patches:
                 continue
-
+            
             if not self.message_dao.get(message.in_reply_to):
                 continue
-
+            
             # Reply is a patch to be uploaded (as the parent of patchset is not in new_messages)
             if message.is_patch():
                 messages_to_upload.append(message.id)
             # Reply is a comment that's parent is not in this batch of messages. Its parent's comments should be reuploaded
             else:
                 messages_with_new_comments.add(message.in_reply_to)
-
+        
         self.upload_messages(messages_to_upload)
-
+            
         self.upload_comments(messages_with_new_comments)
-
+        
         self.remove_files(FILE_DIR)
-
+    
     def update_message_dir(self) -> List[Message]:
         self.last_hash = archive_updater.fill_message_directory(GIT_PATH, FILE_DIR, self.last_hash)
-        messages = self.archive_index.update(FILE_DIR)
+        messages = self.archive_index.update(FILE_DIR) 
         return messages
-
-    # TODO(@willliu): change print to logging and add logs for # of messages uploaded and # that should be uploaded
+    
     def upload_messages(self, messages_to_upload : List[str]):
+        failed = 0
+        subject = 'Temp Subject'
```

Comment:
```suggestion
        subject = '<message not found>`
```

this should make it clearer if the issue was that we couldn't find the patch.

But alternatively, I'd be fine with us just logging the message id instead of the subject.
In this case, they're about as equally useful in looking up the offending email, with the message id being a bit quicker to lookup on lore + the git archives.

Reply:
ok ok, i'll just log the message id then